service/history/execution/mutable_state_builder_methods_activity_test.go

```diff
@@ -191,7 +192,255 @@ func Test__AddActivityTaskCompletedEvent(t *testing.T) {
 		mb.pendingActivityIDToEventID["1"] = 1
 		mb.updateActivityInfos[1] = ai
 		mb.hBuilder = NewHistoryBuilder(mb)
-		_, err := mb.AddActivityTaskCompletedEvent(1, 1, request)
+		event, err := mb.AddActivityTaskCompletedEvent(1, 1, request)
+		assert.NoError(t, err)
+		assert.Equal(t, int64(1), event.ActivityTaskCompletedEventAttributes.ScheduledEventID)
+	})
+}
+
+func Test__tryDispatchActivityTask(t *testing.T) {
+	mb := testMutableStateBuilder(t)
+	event := &types.HistoryEvent{}
+	ai := &persistence.ActivityInfo{}
+	result := mb.tryDispatchActivityTask(context.Background(), event, ai)
+	assert.True(t, result)
+}
+
+func Test__ReplicateActivityTaskCanceledEvent(t *testing.T) {
+	mb := testMutableStateBuilder(t)
+	event := &types.HistoryEvent{
+		EventType: types.EventTypeActivityTaskCanceled.Ptr(),
+		ActivityTaskCanceledEventAttributes: &types.ActivityTaskCanceledEventAttributes{
+			ScheduledEventID: 1,
+		},
+	}
+	ai := &persistence.ActivityInfo{
+		ActivityID: "1",
+	}
+	mb.pendingActivityInfoIDs[1] = ai
+	mb.pendingActivityIDToEventID["1"] = 1
+	err := mb.ReplicateActivityTaskCanceledEvent(event)
```

Comment:
I'd expect some change in state in `mb` ? Anything we could assert? (Same for most of the tests)